app/boards/arm/bt60/bt60.dtsi

```diff
@@ -0,0 +1,128 @@
+/*
+* Copyright (c) 2021 Polarity Works
+*
+* SPDX-License-Identifier: MIT
+*/
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include <dt-bindings/zmk/matrix_transform.h>
+
+/ {
+	model = "BT60";
+	compatible = "bt60";
```

Comment:
The compatible property should be in the form `<manufacturer>,<model>`. Maybe `pw,bt60` or `polarityworks,bt60`?